### PR TITLE
Frontend fixes

### PR DESCRIPTION
**Pull request by Karinon**
This PR adresses the following issues:

- No hardcoded categories anymore. Every found category will be displayed as it is with a capital first letter. If there is no category, then a new one, "others", will be created. All categories are now sorted by alphabet except for "others" which goes to the bottom. See screenshots.
- MENU_ENTRIES can now be (relative) paths which do not lead to django-templates (e.g. flatpages)
- front-page-texts now look less ugly on mobile devices
- updated node-dependencies
- fixed minor dev-env issue where the React-Parts constantly lost the connection to the npm-asset-server 

somestuffimadeup
uncommented
visu

### CODE REVIEW

File: django_evaluation/settings/local.py [hunk 1]
```diff
@@ -310,6 +310,10 @@ def _set_favicon(html_color: str, project_root: Path) -> None:
     "variable",
 ]
 MENU_ENTRIES = []
+
+# Sometimes it is desired to put a link into the navbar which does not correspond
+# to a template inside django. If something like this is needed, put a slash as a
+# prefix to your relative url (the second value in each of the lists), e.g. "/impressum"
```
Comment (antarcticrainforest) [inline django_evaluation/settings/local.py:316]: Do we have a docu for the web, I believe not - correct?

Comment (Karinon) [reply django_evaluation/settings/local.py:316]: Nope.

Comment (antarcticrainforest) [reply django_evaluation/settings/local.py:316]: K, but could add a an example into the README?

Comment (Karinon) [reply django_evaluation/settings/local.py:316]: For what? The Menu_entries? I mean, I could, but that would appear a bit random to me to explain this little thingy and leave everything else as it is. There are things in the web going on that are way more harder to grasp than this and are still not documented. If you want more documentation perhaps we should tackle that in a separate PR (and clean up the settings while we are at it).

Comment (antarcticrainforest) [reply django_evaluation/settings/local.py:316]: ok ok

**Review by antarcticrainforest — APPROVED** [review]
LGTM

**Pull request merged**